server.go

```diff
@@ -1566,7 +1570,9 @@ func (s *Server) Serve(ln net.Listener) error {
 			//
 			// There is a hope other servers didn't reach their
 			// concurrency limits yet :)
-			time.Sleep(100 * time.Millisecond)
+			if !s.DisableSleepWhenConcurrencyLimitsExceeded {
+				time.Sleep(100 * time.Millisecond)
```

Comment:
Can you switch this around and make the option `SleepWhenConcurrencyLimitsExceeded` and make it a `time.Duration` of how long to sleep (with 0 obviously no sleep at all). I think the default of not sleeping at all is better and making it possible to configure how long to sleep is also better.

Reply:
Fixed.

But we cannot do `0` as "no sleep at all" here. Server is initialized directly (without any factory or whatever) so I cannot set a default value for `SleepWhenConcurrencyLimitsExceeded`. So `0` will be the default value. And I think we don't want to change the default behavior. So I made `0` to be overrided by a real default value `100ms`. And to disable the sleeping it's required to set any negative value.

Reply:
That's what I mean, I want to change the default behavior as for most people it doesn't make sense. The default behavior now only makes sense if you have multiple processes listening on the same port. Most people don't even know how to set this up and won't be using a setup like this.

Reply:
Ah, nice. Fixed :)